src/utils/__tests__/style-formatter.spec.js

```diff
@@ -232,15 +232,15 @@ describe('style-formatter', () => {
       expect(text.style.justifyContent).to.equal('center');
     });
 
-    it('should set fontSize when text offsetWidth is bigger than button', () => {
```

Comment:
I don't fully understand what this test tested before, do we need another one to fix this?

Reply:
It is the third step of the font size calculations, we check if the width of the text is too wide to fit. Then we scale it according to the width instead.